CONTRIBUTING.md

```diff
@@ -100,6 +100,7 @@ _… and thanks for contributing!_
 ### Copyright & Licenses
 
 Copyright © 2016-2024 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
```

Comment:
```suggestion
Copyright © 2016-2025 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
```

Reply:
Or maybe we should change this to
```
Copyright © 2016 and later: Unicode, Inc. Unicode and ...
```

@srl295 FYI

Reply:
Done (changed to 2016 and later)